### PR TITLE
[ActionSheet] Promote theming extension to Ready

### PR DESCRIPTION
**Pull request by codeman7**
## Related links
* Component: [ActionSheet](https://github.com/material-components/material-components-ios/tree/develop/components/ActionSheet)

## Introduction
This PR promotes the theming extension for MDCActionSheet from Beta to ready. This has been in beta for over a month and a half and has the minimum client requirements. 

## The fix
Add this subspec to the material components podspec.

### CODE REVIEW

File: catalog/Podfile [hunk 1]
```diff
@@ -60,7 +60,6 @@ target "MDCCatalog" do
   pod 'MaterialComponentsBeta', :path => '../', :testspecs => [
     'ActionSheet+ActionSheetThemer/UnitTests',
     'ActionSheet+ColorThemer/UnitTests',
-    'ActionSheet+Theming/UnitTests',
```
Comment (jverkoey) [inline catalog/Podfile:63]: Why are we removing the test targets?

Comment (codeman7) [reply catalog/Podfile:63]: I put it with the component tests. Was that not what we intended? I guess my question is do we want test subspecs for the theming targets?

Comment (jverkoey) [reply catalog/Podfile:63]: The theming extensions are still technically separate targets so I think it’s valuable for the test targets to match; reduces likelihood of our tests crossing concerns between separate targets.

Comment (codeman7) [reply catalog/Podfile:63]: I can agree with that, I have reverted this change and the move.

**Comment by jverkoey** [comment]
Closing this PR due to inactivity. Please re-open when you intend to pursue it.

**Pull request closed without merging**